worldedit-core/src/main/java/com/sk89q/worldedit/world/World.java

```diff
@@ -117,7 +141,20 @@
      * @param previousType the type of the previous block that was there
      * @return true if the block was successfully notified
      */
-    boolean notifyAndLightBlock(BlockVector3 position, BlockState previousType) throws WorldEditException;
+    @Deprecated
+    default boolean notifyAndLightBlock(BlockVector3 position, BlockState previousType) throws WorldEditException {
+        return applySideEffects(position, previousType, SideEffectSet.defaults());
+    }
+
+    /**
+     * Applies a set of side effects on the given block.
+     *
+     * @param position position of the block
+     * @param previousType the type of the previous block that was there
+     * @param sideEffectSet which side effects to perform
+     * @return true if the block was successfully notified
```

Comment:
This still says `notified` rather than ... `@return true if the side effects were successfully applied`.

Thinking about API design here, would it make more sense to (1) use a normal `Set<SideEffect>` here, as the specific state of each effect has no bearing on the action taken by `applySideEffects`; and (2) return a `Set<SideEffect>` of the effects that were actually applied OR not able to be applied (not sure which is better). It just seems like a `boolean` is not expressive enough when a platform could implement _some_ effects but not all.